data/SAbDab_Chen/meta.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name: SAbDab_Chen
+description: "Antibody data from Chen et al, where they process from the SAbDab. \n        From an initial dataset of 3816 antibodies, they retained 2426\
+    \ antibodies\n        that satisfy the following criteria: 1. \n        have both sequence (FASTA) and Protein Data Bank (PDB) structure files,\n  \
+    \      2. contain both a heavy chain and a light chain, and 3. \n        have crystal structures with resolution < 3 Å. \n        The DI label is derived\
+    \ from BIOVIA's pipelines."
+targets:
+    - id: developability
+      description: functional antibody candidate to be developed into a manufacturable(1), or not(0)
+      units: ''
+      type: categorical
+      names:
+          - antibody developability
+          - monoclonal anitbody
+          - functional antibody candidate
+          - manufacturable, stable, safe, and effective antibody drug
+      uris:
+          - https://rb.gy/idkdqp
+          - https://rb.gy/b8cx8i
```

Comment:
With URIs we mean links to ontologies, such as the ones you can find here https://bioportal.bioontology.org/ontologies/BAO?p=classes&conceptid=http://purl.obolibrary.org/obo/NCIT_C20604

Reply:
I removed those as they were not fitting our setup.